packages/stryker/src/transpiler/CoverageInstrumenterTranspiler.ts

```diff
@@ -30,9 +30,9 @@ export default class CoverageInstrumenterTranspiler implements Transpiler {
         outputFiles: files.map(file => this.instrumentFileIfNeeded(file)),
         error: null
       };
-      return this.addCollectCoverageFileIfNeeded(result);
+      return Promise.resolve(this.addCollectCoverageFileIfNeeded(result));
     } catch (error) {
-      return this.errorResult(errorToString(error));
+      return Promise.resolve(this.errorResult(errorToString(error)));
     }
```

Comment:
We might want to leave the function `async` because otherwise the two return statements have to be a `Promise`.

Reply:
I think this is fine as it is. It looks clean in the resulting JavaScript.